db/migrate/20210524085125_create_client_promotions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateClientPromotions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :client_promotions do |t|
+      t.integer :promotions_id
```

Comment:
Always when you try to create association in database, please use foreign keys or references (I can recommend this one)

So it would be something like this:
```ruby
class CreateClientPromotions < ActiveRecord::Migration[6.1]
  def change
    create_table :client_promotions do |t|
      t.references :promotions
      t.references :clients

      t.timestamps
    end
  end
end
```

https://guides.rubyonrails.org/association_basics.html

Please check difference between column and reference [here](https://rubyinrails.com/2019/04/22/rails-add-reference-vs-add-column-migration/)

Consider this change in all migrations. It will make your life easier.

Reply:
btw. I've just realized that you changed it in some tables. Try to redefine it for all tables to make it consistent 👍

Reply:
yeah, thanks to you I just realized that I didn't change one table, need to fix it.